content/en/docs/setup/independent/control-plane-flags.md

```diff
@@ -22,8 +22,10 @@ The `extraArgs` field consist of `key: value` pairs. To override a flag for a co
 1.  Add the appropriate fields to your configuration.
 2.  Add the flags to override to the field.
 
+You can also add `extraVolumes` to mount additional volumes if necessary (e.g. including configuration files that are relevant to your cloud provider).
+
 For more details on each field in the configuration you can navigate to our
-[API reference pages](https://godoc.org/k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm#ClusterConfiguration).
+[API reference pages](https://godoc.org/k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1beta1#hdr-Kubeadm_init_configuration_types).
```

Comment:
should be only `[API reference pages](https://godoc.org/k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1beta1)`